volontulo/templates/organizations/organization_view.html

```diff
@@ -6,19 +6,19 @@
     <h2>Organizacja</h2>
     <div class="form-horizontal">
         <div class="form-group form-group-sm">
-            <label class="col-xs-2 control-label" for="{{ organization_form.name.id_for_label }}">Nazwa</label>
+            <label class="col-xs-2 control-label">Nazwa</label>
             <div class="col-xs-4 form-control-static">
                 <span>{{ organization.name }}</span>
             </div>
-            <label class="col-xs-2 control-label" for="{{ organization_form.address.id_for_label }}">Adres</label>
+            <label class="col-xs-2 control-label">Adres</label>
             <div class="col-xs-4 form-control-static">
                 <span>{{ organization.address }}</span>
             </div>
         </div>
         <div class="form-group form-group-sm">
-            <label class="col-xs-2 control-label" for="{{ organization_form.description.id_for_label }}">Opis</label>
-            <div class="col-xs-10 form-control-static">
-                <span>{{ organization.description }}</span>
+            <label class="col-xs-2 control-label">Opis</label>
+            <div class="col-xs-10">
```

Comment:
you have missed `form-control-static` class there. It is needed so label aligns well with the value